pkg/scheduler/schedule_one.go

```diff
@@ -823,14 +823,17 @@ func (sched *Scheduler) handleSchedulingFailure(fwk framework.Framework, podInfo
 	pod := podInfo.Pod
 	msg := truncateMessage(err.Error())
 	fwk.EventRecorder().Eventf(pod, nil, v1.EventTypeWarning, "FailedScheduling", "Scheduling", msg)
-	if err := updatePod(sched.client, pod, &v1.PodCondition{
-		Type:    v1.PodScheduled,
-		Status:  v1.ConditionFalse,
-		Reason:  reason,
-		Message: err.Error(),
-	}, nominatingInfo); err != nil {
-		klog.ErrorS(err, "Error updating pod", "pod", klog.KObj(pod))
-	}
+	go func() {
+		// update Pod's status in goroutine since the updating may take a time by retries.
```

Comment:
```suggestion
		// update Pod's status in goroutine because there might be retries.
```

Reply:
But let's do this just for calling `util.PatchPodStatus`

Reply:
Sure. Updated.